railties/test/application/configuration_test.rb

```diff
@@ -1209,6 +1209,15 @@ def create
       assert_equal false, ActionController::Parameters.action_on_unpermitted_parameters
     end
 
+    test "config.action_controller.default_protect_from_forgery is true by default on production" do
+      app "production"
```

Comment:
The rest of the tests in this file set this as "development" when the behaviour doesn't depend on the particular environment; we should probably do the same here for consistency.